### PR TITLE
Balance/buff specialist pouches

### PR DESCRIPTION
**Pull request by null-Aurelian**
## About The Pull Request
Buffs the some specialist pouches even more to encourage use in lieu of the de-facto best in slot medium pouch. Converts several of these pouches to use the continuous storage system, bringing them in line with generic pouches. Overall the pouches hold approximately the same number of objects or less but the more freeform storage capacity should provide options for players.

## Why It's Good For The Game
Currently people only really use the medium generic pouch, so these changes should entice people to use the other pouches. Those that weren't changed to the new system typically had other changes, such as the Engineering Tool Pouch being made into basically a miniature Tool Belt.

## Changelog
:cl:
balance: Medical Supply Pouch converted to use continuous storage, equivalent to the medium pouch in space. Medical pouch can now hold nanopaste.
balance: Engineering Tool Pouch slots increased by 1 (3 to 4), can hold normal sized items and nanopaste. 
balance: Engineering Supply Pouch converted to use continuous storage, equivalent to a large pouch in space.
balance: Engineering Material Pouch converted to use continuous storage, enough space for 3 stacks of materials
balance: Gun Part Pouch converted to use continuous storage, enough space for 8 normal parts and can now hold tool upgrade mods
/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/storage/pouches.dm [hunk 1]
```diff
@@ -103,18 +104,19 @@
 		/obj/item/clothing/gloves/latex,
 		/obj/item/reagent_containers/hypospray,
 		/obj/item/clothing/glasses/hud/health,
+		/obj/item/stack/nanopaste
 		)
 
 /obj/item/storage/pouch/engineering_tools
 	name = "engineering tools pouch"
-	desc = "Can hold small engineering tools. But only about three pieces of them."
+	desc = "A pouch for holding Engineering tools. Looks like there are pockets in it for 4 tools."
```
Comment (Firefox13) [inline code/game/objects/items/weapons/storage/pouches.dm:112]:
```suggestion
	desc = "A pouch for holding engineering tools. Looks like there are pockets in it for 4 tools."
```

File: code/game/objects/items/weapons/storage/pouches.dm [hunk 2]
```diff
@@ -135,18 +137,20 @@
...
 		/obj/item/cell/medium,
-		/obj/item/gun/projectile/flare_gun
+		/obj/item/gun/projectile/flare_gun,
+		/obj/item/stack/nanopaste
 		)
 
 /obj/item/storage/pouch/engineering_supply
 	name = "engineering supply pouch"
-	desc = "Can hold engineering equipment. 12 pieces of hardware, cells, rods or cables."
+	desc = "A pouch for holding various engineering scanners, powercells and equipment."
```
Comment (Firefox13) [inline code/game/objects/items/weapons/storage/pouches.dm:146]:
```suggestion
	desc = "A pouch for holding various engineering scanners, power cells and equipment."
```

File: code/game/objects/items/weapons/storage/pouches.dm [hunk 3]
```diff
@@ -339,16 +344,18 @@
...
 	name = "part pouch"
-	desc = "Can hold gun parts and armor parts."
+	desc = "A pouch for holding all sorts of small parts, upgrades and components."
 	icon_state = "part_pouch"
 	item_state = "part_pouch"
 	rarity_value = 33
 
-	storage_slots = 10
+	storage_slots = null
+	max_storage_space = DEFAULT_NORMAL_STORAGE * 0.8 //Actually smaller than previous but illusion of space with continuous holding space
```
Comment (Firefox13) [inline code/game/objects/items/weapons/storage/pouches.dm:353]: lol, okay

**Comment by MLGTASTICa** [comment]
Overbuffing pouches instead of nerfing the medium general pouch. Can't wait to become joe rambo as NT because i have an ammo pouch , med pouch and a toolbelt with enough space to hold a entire armoury worth of ammo , a medbay of medicine and   enough tools to break through any wall or door.

**Comment by null-Aurelian** [comment]
Honestly, if it is an overbuff, we can hold off on this  since this isn't a time critical thing
I do think mediums only being able to hold small items would be better but it is much harder to convince people that nerfs should be done.

**Comment by null-Aurelian** [comment]
@hyperioo , @Firefox13 can we hold off on merging this while more data is gathered?

**Comment by Firefox13** [comment]
> can we hold off on merging this while more data is gathered?

I'd say the buffs are fine, it's just SPCR being salty as usual.

**Comment by assortedbeads** [comment]
SPCR coping on PRs again, what else is new.

If someone really wanted all that they would just carry a duffel bag or something, this isnt gonna make a "pouch meta" or whatever he's suggesting.

**Comment by hyperioo** [comment]
> @hyperioo , @Firefox13 can we hold off on merging this while more data is gathered?

Sure, we can test it during a test round next week

**Review by hyperioo — APPROVED** [review]

**Comment by MLGTASTICa** [comment]
Balance ruined

**Pull request merged**